Libraries/CommonFunctions.psm1

```diff
@@ -647,9 +647,9 @@ function Install-CustomLIS ($CustomLIS, $customLISBranch, $allVMData, [switch]$R
 	try
 	{
 		$CustomLIS = $CustomLIS.Trim()
-		if ( ($CustomLIS -ne "lisnext") -and !($CustomLIS.EndsWith("tar.gz")))
+		if ( ($CustomLIS -ne "lisnext") -and !($CustomLIS.EndsWith("tar.gz")) -and ($CustomLIS -ne "latest"))
```

Comment:
Use meaningful option. Use LatestLIS instead of Latest.

Reply:
changed to LatestLIS